src/main/java/no/digipost/signature/client/portal/SignatureStatus.java

```diff
@@ -77,6 +78,18 @@
      */
     public static final SignatureStatus BLOCKED = new SignatureStatus("BLOCKED");
 
+    /**
+     * Indicates that the service was unable to retrieve the signer's name.
+     * <p>
```

Comment:
Samme for beskrivelse her angående å prøve på nytt som i API-doc om det gir mening. Ellers er jeg fornøyd i alle fall!

Reply:
Ref https://github.com/digipost/signature-api-specification/pull/70#discussion_r150529597 gir det ikke mening å prøve på nytt i dette tilfellet :)